### PR TITLE
[7.3] ImplodeCallFixer - add tests for PHP7.3

### PR DESCRIPTION
**Pull request by kubawerlos**

### CODE REVIEW

File: tests/Fixer/FunctionNotation/ImplodeCallFixerTest.php [hunk 1]
```diff
@@ -146,4 +146,44 @@ public function provideFixCases()
...
+     * @dataProvider provideFix73Cases
+     */
+    public function testFix73($expected, $input = null)
+    {
+        $this->doTest($expected, $input);
+    }
+
+    public function provideFix73Cases()
+    {
+        yield ['<?php implode("", $foo, );'];
```
Comment (keradus) [inline tests/Fixer/FunctionNotation/ImplodeCallFixerTest.php]: i don't think we need to yield static array that is not generated

Comment (keradus) [inline tests/Fixer/FunctionNotation/ImplodeCallFixerTest.php]: this one is provided by next case as well

File: tests/Fixer/FunctionNotation/ImplodeCallFixerTest.php [hunk 2]
```diff
@@ -146,4 +146,42 @@ public function provideFixCases()
...
+     * @dataProvider provideFix73Cases
+     */
+    public function testFix73($expected, $input)
+    {
+        $this->doTest($expected, $input);
+    }
+
+    public function provideFix73Cases()
+    {
+        yield [
```
Comment (keradus) [inline tests/Fixer/FunctionNotation/ImplodeCallFixerTest.php]: i don't think we need to yield static array that is not generated

Comment (kubawerlos) [reply tests/Fixer/FunctionNotation/ImplodeCallFixerTest.php]: Original tests were with `yield` (made by me :smile:) so I continued with it - it makes one level of array less in data providers.

**Comment by keradus** [comment]
Thank you @kubawerlos.

File: tests/Fixer/FunctionNotation/ImplodeCallFixerTest.php (hunk 2 above)
Comment (TomasVotruba) [reply tests/Fixer/FunctionNotation/ImplodeCallFixerTest.php]: Better make it consistent thought, one way, or another.

There is Rector for yield approach - https://github.com/rectorphp/rector/blob/master/config/level/phpunit/phpunit-yield-data-provider.yml

Comment (kubawerlos) [reply tests/Fixer/FunctionNotation/ImplodeCallFixerTest.php]: OMG, I've wasted a lot of time to change some nasty data providers manually...

Is there maybe a Rector to change data providers to [static](https://github.com/sebastianbergmann/phpunit/blob/7.5.1/src/Util/Test.php#L862) methods?

Comment (keradus) [reply tests/Fixer/FunctionNotation/ImplodeCallFixerTest.php]: no statics, please...

Comment (TomasVotruba) [reply tests/Fixer/FunctionNotation/ImplodeCallFixerTest.php]: @kubawerlos No more manual labor for programmers :D I'm not sure what you mean. Create issues at Rector with example and I'll see what we can do.

**Pull request merged**